content/en/continuous_integration/setup_pipelines/circleci.md

```diff
@@ -29,6 +29,8 @@ The Datadog integration for [CircleCI][1] works by using [webhooks][2] to send d
 
 2. Click **Add Webhook** to save the new webhook.
 
+To set custom tags, add to the **Webhook URL** a query parameter `tags`, with *key:value* pairs separated by commas. If the tag contains commas, quotes are needed. Example: `key1:value1,"key2: value with , comma",key3:value3`. (**This parameter needs URL encoding**, for the example: `?tags=key1%3Avalue1%2C%22key2%3A+value+with+%2C+comma%22%2Ckey3%3Avalue3`)
```

Comment:
```suggestion
To set custom tags, add to the **Webhook URL** a URL-encoded query parameter `tags`, with `key:value` pairs separated by commas. If the tag pair contains any commas, quotes are needed. For example, to add `key1:value1,"key2: value with , comma",key3:value3`, the following string would need to be appended to the 
**Webhook URL**: `?tags=key1%3Avalue1%2C%22key2%3A+value+with+%2C+comma%22%2Ckey3%3Avalue3`
```

Maybe this reads better? I felt like the URL encoding should be raised at the beginning